addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -660,15 +660,16 @@ def _import_fill_invoice_line_form(self, tree, invoice_line, qty_factor):
         tax_nodes = tree.findall('.//{*}Item/{*}ClassifiedTaxCategory/{*}Percent')
         if not tax_nodes:
             for elem in tree.findall('.//{*}TaxTotal'):
-                tax_nodes += elem.findall('.//{*}TaxSubtotal/{*}Percent')
+                tax_node = elem.findall('.//{*}TaxSubtotal//{*}Percent')
+                tax_nodes += tax_node
```

Comment:
useless diff

Reply:
Make it back to what it was and try to make the test pass without this. 😁

Reply:
We decided to put that part in another commit as it's a fix by itself for UBL 2.0